src/Makefile

```diff
@@ -95,9 +95,13 @@ FINAL_LDFLAGS=$(LDFLAGS) $(REDIS_LDFLAGS) $(DEBUG)
 FINAL_LIBS=-lm
 DEBUG=-g -ggdb
 
-# Linux ARM32 needs -latomic at linking time
+# Linux ARM needs -latomic at linking time
+ifneq (,$(filter aarch64 armv,$(uname_M)))
```

Comment:
`aarch64` surely does not need `libatomic`. Same for `armv7` and even possibly not required if building for `armv6k` target.

It does not seem to fix anything or am I missing something? Or are you cross-compiling `redis` on `aarch64` host for `armv6` in which case you should not depend on `uname -m` and just override it for target arch `make uname_M=armv6l`.

Reply:
Thank you for explaining this. We are indeed cross-compiling redis on aarch64 hosts for armv6, [issue](https://gitlab.com/gitlab-org/omnibus-gitlab/-/issues/6605). Will move forward with the `uname_M` make flag. Closing this PR.